lib/cdo/script_constants.rb

```diff
@@ -18,15 +18,9 @@ module ScriptConstants
   # the category it belongs to in course dropdowns. The order of scripts within
   # a category will be the order in which they appear in the dropdown, and the
   # order of the categories will be their order in the dropdown.
+  # Note: This is only for scripts. Courses that have a family_name will always come first.
```

Comment:
I like the detail you're adding, but you might want to note that this sentence (or perhaps this entire comment block?) only applies to the AssignmentSelector within EditSectionForm. In other cases, for example the script selector in the progress tab, full courses do not always come first:
<img width="519" alt="Screen Shot 2019-06-03 at 5 12 44 PM" src="https://user-images.githubusercontent.com/8001765/58842421-f5103180-8622-11e9-92cd-04acba9e0ae4.png">

Reply:
ohh, good catch! i'll remove the first part of this comment to avoid confusion, but leave the part about checking `ScriptConstants#category_priority` for more detail.